src/control/lib/telemetry/promexp/collector.go

```diff
@@ -113,6 +113,25 @@ func sanitizeMetricName(in string) string {
 	}, strings.TrimLeft(in, "/"))
 }
 
+func parseNameSubstr(labels labelMap, name string, matchRE string, replacement string,
+	assignLabels func(labelMap, []string)) string {
```

Comment:
nit: re consistency, thought we were leaving function signatures long

Reply:
Fixed